integrations/integration.php

```diff
@@ -1,66 +1,298 @@
 <?php
+/**
+ * Base class for Integration.
+ *
+ * @package Automattic\VIP\Integrations
+ */
 
 namespace Automattic\VIP\Integrations;
 
+use InvalidArgumentException;
+
+// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound -- Disabling due to enums.
+
+/**
+ * Enum which represent all possible statuses for the client integration via VIP.
+ *
+ * These should be in sync with the statuses available on the backend.
+ */
+abstract class Client_Integration_Status {
+	const BLOCKED = 'blocked';
+}
+
+/**
+ * Enum which represent all possible statuses for the site integration via VIP.
+ *
+ * These should be in sync with the statuses available on the backend.
+ */
+abstract class Site_Integration_Status {
+	const ENABLED  = 'enabled';
+	const DISABLED = 'disabled';
+	const BLOCKED  = 'blocked';
+}
```

Comment:
I think that will cause confusion while reading code because client only have support for `BLOCKED` status or nothing. The current implementation of enums aligns with that.